providers/catalog.go

```diff
@@ -1215,4 +1216,28 @@
 			Write:     false,
 		},
 	},
+
+	Typeform: {
+		AuthType: Oauth2,
+		BaseURL:  "https://api.{{.dataCenter}}typeform.com",
```

Comment:
Can you please update this to `api.typeform.com`, and add a separate `TypeformEU` connector?

Reply:
Added